drivers/node/ssh/ssh.go

```diff
@@ -377,6 +411,39 @@ func (s *SSH) InjectNetworkError(nodes []node.Node, errorInjectionType string, o
 	return nil
 }
 
+// InjectNetworkErrorWithRebootFallback by dropping packets or introdiucing delay in packet tramission and reboot nodes during fallback
+// nodes=> list of nodes where network injection should be done.
+// errorInjectionType => pass "delay" or "drop"
+// operationType => add/change/delete
+// dropPercentage => intger value from 1 to 100
+// delayInMilliseconds => 1 to 1000
+
+// sample usage:
+// Inst().N.InjectNetworkErrorWithRebootFallback(nodes, "delay", "del", 0, i)
+func (s *SSH) InjectNetworkErrorWithRebootFallback(nodes []node.Node, errorInjectionType string, operationType string, dropPercentage int, delayInMilliseconds int) error {
```

Comment:
Create a type for `errorInjectionType` and `operationType`

Reply:
The would require changes to existing parent function InjectNetworkError